app/javascript/Components/Result/dropdown_menu.jsx

```diff
@@ -0,0 +1,55 @@
+import React from "react";
+import PropTypes from "prop-types";
+
+import safe_marked from "../../common/safe_marked";
+
+export class DropDownMenu extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {onHover: false};
+  }
+
+  handleMouseEnter = () => {
+    this.setState({onHover: true});
+  };
+
+  handleMouseLeave = () => {
+    this.setState({onHover: false});
+  };
+
+  render() {
+    return (
+      <li
+        className="dropdown_menu"
```

Comment:
It's great you are changing the class name to make this component more generic, but in this case the class name was likely used for styling purposes. So, by changing this you also need to change the CSS in order to preserve the styling.